src/main/java/strategy/ReturnOperationStrategy.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
+
+import java.util.Map;
+
+public class ReturnOperationStrategy implements OperationStrategy {
+    @Override
+    public void execute(String fruit, int quantity, Map<String, Integer> fruitQuantities) {
+        fruitQuantities.put(fruit, fruitQuantities.getOrDefault(fruit, 0) + quantity);
```

Comment:
please check usage of .merge() method instead of .put()
here it should look more pretty